tools/csv-merger/csv-merger.go

```diff
@@ -534,6 +536,8 @@ func getDeploymentParams() *components.DeploymentOperatorParams {
 		VirtIOWinContainer: *kvVirtIOWinImage,
 		Smbios:             *smbios,
 		Machinetype:        *machinetype,
+		Amd64MachineType:   *amd64MachineType,
+		Arm64MachineType:   *arm64MachineType,
```

Comment:
You are adding the `Amd64MachineType` and `Arm64MachineType`, but I don't see them defined in https://github.com/tiraboschi/hyperconverged-cluster-operator/blob/3585d39f79f59024d7b1b3fef11055b5ef99ab3f/pkg/components/components.go#L65

Reply:
I don't know to which version GitHub took me, but I do see them on `main`

Reply:
They come with: https://github.com/tiraboschi/hyperconverged-cluster-operator/commit/35a84c89be94acfa43e8a867a9f0a169d0a8d3d0#diff-f1531df30500812ec42a33a52023c9ceca8a93d4972132fd241ad143b107c063R79-R80